index.html

```diff
@@ -3,9 +3,6 @@
 ---
 
 <div class="home">
-
-  <h1 class="page-heading">Posts</h1>
```

Comment:
How are we supposed to differentiate this content group from all the other content types on the page now?! :trollface: